.github/workflows/ci.yml

```
@@ -0,0 +1,44 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php: [7.3, 7.4, 8.0]
+        libraries:
+          - laravel: ^8.0
+            testbench: ^6.0
+          - laravel: ^7.0
+            testbench: ^5.0
+          - laravel: ^6.0
+            testbench: ^4.0
```

Write them in one line 

```suggestion
          - { laravel: ^8.0, testbench: ^6.0, composer_flags: --prefer-stable }
          - { laravel: ^7.0, testbench: ^5.0, composer_flags: --prefer-stable }
          - { laravel: ^6.0, testbench: ^4.0, composer_flags: --prefer-stable }
          - { laravel: ^6.0, testbench: ^4.0, composer_flags: --prefer-stable --prefer-lowest }
```